mockserver/update_state.py

```diff
@@ -156,5 +167,5 @@
 
 # Save
 with open('mockserver/public/v3/state', 'w') as f:
-    json.dump({'data': obj}, f)
+    json.dump({'data': obj}, f, indent=4, sort_keys=True)
```

Comment:
will make it easier to see diffs in github

Reply:
Great idea!